src/app/routes/homePage/getInitialData/index.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { Agent } from 'https';
+import Url from 'url-parse';
+import nodeLogger from '../../../lib/logger.node';
+import { BFF_FETCH_ERROR } from '../../../lib/logger.const';
+import fetchPageData from '../../utils/fetchPageData';
+import { Services } from '../../../models/types/global';
+import HOME_PAGE_CONFIG from './page-config';
+
+const logger = nodeLogger(__filename);
+
+type Envs = 'test' | 'live' | 'local' | 'undefined';
+
+const getEnvironment = (pathname: string) => {
+  if (pathname.includes('renderer_env=test')) {
+    return 'test';
+  }
+  if (pathname.includes('renderer_env=live')) {
+    return 'live';
+  }
+
+  return process.env.SIMORGH_APP_ENV as Envs;
+};
```

Comment:
Can we extract this into a separate utility within  src/app/routes/utils/getEnvironment, so that we can test it, and use it elsewhere e.g. in getInitialData for article + topic?

Reply:
We can leave this for later and address as part of https://github.com/bbc/simorgh/pull/10683